numpy/lib/function_base.py

```diff
@@ -1094,6 +1094,18 @@ def diff(a, n=1, axis=-1):
     axis : int, optional
         The axis along which the difference is taken, default is the
         last axis.
+    prepend : array_like
+        Values to prepend to the beginning of "a" along axis before
```

Comment:
I think it's worth noting that this only exists as a more efficient option than `np.concatenate`